bin/tachyon-workers.sh

```diff
@@ -15,7 +15,7 @@ if [ $# -le 0 ]; then
 fi
 
 bin=`cd "$( dirname "$0" )"; pwd`
-DEFAULT_LIBEXEC_DIR="$bin"/../libexec
```

Comment:
the second parameter can be either "", "local", or "worker". When it is "workers", the init_env function is not invoked si the variable is not set.